tests/device/get_info.cpp

```diff
@@ -0,0 +1,21 @@
+/* RUN: %{execute}%s
+*/
+#include <CL/sycl.hpp>
+
+#include <boost/test/minimal.hpp>
+
+using namespace cl::sycl;
+
+int test_main(int argc, char *argv[]) {
+  // Get all the devices
+  auto devices = device::get_devices();
+  // Verify that the device_type returned match what is expected
+  for (auto const &d : devices) {
+    BOOST_CHECK(d.get_info<info::device::max_work_group_size>()>0);
```

Comment:
With the real value it would be better.
Also a test missing for ``device::get_info<info::device::device_type>()``